schema/pgsql/upgrades/026.sql

```diff
@@ -8,4 +8,5 @@ ALTER TABLE contact ALTER COLUMN username TYPE varchar(254);
 ALTER TABLE contact_address ALTER COLUMN type TYPE varchar(255);
 ALTER TABLE object_id_tag ALTER COLUMN tag TYPE varchar(255);
 ALTER TABLE object_extra_tag ALTER COLUMN tag TYPE varchar(255);
+ALTER TABLE browser_session ALTER COLUMN username TYPE varchar(254);
```

Comment:
FYI. 4e39e8b28402928d6160df1e7b9bd9bfa848d94d:

> Schema: make browser_session.username case-sensitive to limit its length
>
> This is equivalent to Icinga Web 2 where the users are case-sensitive
> in Postgres, but not in MySQL. Also the length limitation enables creation
> of an equivalent schema in MySQL. MySQL requires its keys to have a length.

(@nilmerg NOBODY will mix different RDBMS types in Icinga context.)